### DISCUSSION TITLE
Is it possible to gracefully recover from number overflow parsing errors during sax_parse?

### DISCUSSION THREAD

**Discussion by ddlutz-old** [nlohmann/json#3055] [Q&A] [answered]
Background: I am using sax_parse, with the hopes to be able to parse very large numbers that don't fit within C++'s types. I was hoping I would be able to use the parse_error() method, and in there I could custom parse the last_token, and if it matched my large numeric type return true and let json parser continue doing it's thing.

This almost works, except it seems like even if I return 'true' from parse_error, json parse expects end of input and does not continue to parse.

With my simple JSON file of :
```json
{
  "numeric": 1,
  "numeric": 1.0,
  "numeric": 10E1001,
  "another_numeric": 14
}
```

I will end up getting the following from my very simple parser interface (just prints out details as it gets it):

```
parsed key: numeric
error parsing: 10E1001
error details: [json.exception.out_of_range.406] number overflow parsing '10E1001'
error parsing: 10E1001,
error details: [json.exception.parse_error.101] parse error at line 4, column 21: syntax error while parsing value - unexpected ','; expected end of input
```

Is there any way to modify the behavior such that when a "number overflow parsing " error is hit, I can continue parsing that as my own number, and the parser does not expect end of input?

**Comment by nlohmann** [comment, marked as answer]
This seems to be a duplicate of #2834 - I'm afraid this is not possible at the moment, see https://github.com/nlohmann/json/discussions/2834#discussioncomment-1008417.
